co-log/src/Colog/Message.hs

```diff
@@ -139,7 +139,7 @@ types. The type family is open so you can add new instances.
 -}
 type family FieldType (fieldName :: Symbol) :: Type
 type instance FieldType "threadId" = ThreadId
-type instance FieldType "utcTime"  = UTCTime
+type instance FieldType "utcTime"  = C.Time
```

Comment:
According to the documentation for `Time` data type from the `chronos` library, this is POSIX time, not UTC time. 

* https://hackage.haskell.org/package/chronos-1.0.4/docs/Chronos-Types.html#t:Time

So I guess this type family instance should be also changed, from `"utcTime"` to something like `"posixTime"` or `"chronosTime"`.